src/server/service/kv.rs

```diff
@@ -1560,6 +1560,7 @@ fn handle_measures_for_batch_commands(measures: &mut MeasuredBatchResponse) {
             .get(label)
             .get(resource_priority)
             .observe(elapsed.as_secs_f64());
+        GRPC_WAIT_HISTOGRAM.observe(wait.as_secs_f64());
```

Comment:
If the metric is supposed only to cover the batch commands, how about make it explicit in the name, e.g., `GRPC_BATCH_COMMANDS_WAIT_HISTOGRAM`?

Reply:
Good point, addressed.